### PR TITLE
feat: create cser projects page (#2685)

### LINKED ISSUE #2685: [AnVIL Next] Create CSER projects page
#### Approach
* See [existing content](https://cser-consortium.org/projects).
* See [mocks](https://www.figma.com/proto/JOUDXkKfgivRNFdBsWDPEH/AnVIL-Data-Explorer?type=design&node-id=2529-28477&scaling=min-zoom&page-id=2529%3A26617&starting-point-node-id=2529%3A26618&hide-ui=1).
* Projects link to project detail pages (see [existing content](https://cser-consortium.org/projects/155) and [mocks](https://www.figma.com/proto/JOUDXkKfgivRNFdBsWDPEH/AnVIL-Data-Explorer?type=design&node-id=2529-28742&scaling=min-zoom&page-id=2529%3A26617&starting-point-node-id=2529%3A26618&hide-ui=1)).

### PR DESCRIPTION
**Pull request by frano-m**
Closes #2685.

### CODE REVIEW

File: next/components/Consortia/CSER/components/Projects/common/entities.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention -- TODO update eslint rule.
```
Comment (frano-m) [inline next/components/Consortia/CSER/components/Projects/common/entities.ts:1]: @NoopDog, I believe for AnVIL next we need the eslint rule to accept uppercase enum?

File: next/components/common/Figure/figure.styles.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
 import { textBody400 } from "@clevercanary/data-explorer-ui/lib/styles/common/mixins/fonts";
 import styled from "@emotion/styled";
 
-export const ImageWithCaption = styled.div`
+export const Figure = styled.figure`
```
Comment (frano-m) [inline next/components/common/Figure/figure.styles.ts:4]: Refactored the image with caption component - I need a standalone `Image` component...

File: next/components/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+export { Breadcrumbs } from "@clevercanary/data-explorer-ui/lib/components/common/Breadcrumbs/breadcrumbs";
```
Comment (frano-m) [inline next/components/index.tsx:1]: Facilitates easier importing of components!

File: next/docs/common/constants.ts [hunk 4]
```diff
@@ -1,32 +1,25 @@
-import { Breadcrumbs } from "@clevercanary/data-explorer-ui/lib/components/common/Breadcrumbs/breadcrumbs";
-import { Card } from "@clevercanary/data-explorer-ui/lib/components/common/Card/card";
-import { Grid } from "@clevercanary/data-explorer-ui/lib/components/common/Grid/grid";
 import { ANCHOR_TARGET } from "@clevercanary/data-explorer-ui/lib/components/Links/common/entities";
-import { Link } from "@clevercanary/data-explorer-ui/lib/components/Links/components/Link/link";
-import { Image } from "../../components/common/Image/image";
-import { Subheader } from "../../components/common/Typography/components/Subheader/subheader";
-import { BreadcrumbsCSER } from "../../components/Consortia/CSER/components/Breadcrumbs/components/BreadcrumbsCSER/breadcrumbsCSER";
-import { BreadcrumbsNews } from "../../components/Consortia/CSER/components/Breadcrumbs/components/BreadcrumbsNews/breadcrumbsNews";
+import * as C from "../../components";
```
Comment (frano-m) [inline next/docs/common/constants.ts:2]: Single import of all components!

File: next/theme/common/components.ts [hunk 5]
```diff
@@ -30,7 +30,6 @@ export const MuiCssBaseline = (theme: Theme): Components["MuiCssBaseline"] => {
       img: {
         margin: "16px 0",
         maxWidth: "100%",
-        width: "auto",
```
Comment (frano-m) [inline next/theme/common/components.ts:33]: Not required!

**Review by NoopDog — APPROVED** [review]
Wow! ⭐  Nice work @frano-m !

**Pull request merged**